python/ruff-ecosystem/ruff_ecosystem/defaults.py

```diff
@@ -99,6 +93,68 @@
     ),
     Project(
         repo=Repository(owner="indico", name="indico", ref="master"),
+        # Remove once indico removed S401 from their ignore configuration
+        config_overrides={
+            "lint.ignore": [
+                "E226",  # allow omitting whitespace around arithmetic operators
```

Comment:
This is a bit silly but I couldn't find a way to remove a single rule from` ignore`